lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -382,6 +382,10 @@ defmodule ExUnit.Case do
   """
   defmacro describe(message, do: block) do
     quote do
+      if Module.get_attribute(__MODULE__, :describetag) != [] do
+        raise "you can only set @describetag inside a \"describe\" block"
```

Comment:
I don't think this is correct. You can also set a describe tag before a describe block. I think this is a backwards incompatible change.

Reply:
Looks like I'm wrong. But this is already happening here:

https://github.com/elixir-lang/elixir/blob/master/lib/ex_unit/lib/ex_unit/case.ex#L415-L417